src/Application.php

```diff
@@ -201,13 +201,13 @@ public function pipe($path, $middleware = null)
     {
         // Lazy-load middleware from the container when possible
         $container = $this->container;
-        if (null === $middleware && is_string($path) && $container && $container->has($path, true)) {
+        if (null === $middleware && is_string($path) && $container && $container->has($path)) {
```

Comment:
There's nothing to change, @Maks3w ; I made the comment in case somebody wonders what the purpose of that change was. The change is fine.